packages/jsActions/nanoflow-actions-native/src/permissions/RequestGenericPermission.ts

```diff
@@ -0,0 +1,65 @@
+ * @returns {Promise.<"NanoflowCommons.Enum_PermissionStatus.unavailable"|"NanoflowCommons.Enum_PermissionStatus.denied"|"NanoflowCommons.Enum_PermissionStatus.limited"|"NanoflowCommons.Enum_PermissionStatus.granted"|"NanoflowCommons.Enum_PermissionStatus.blocked">}
+ */
+export async function GenericPermission(
+    permission: string
+): Promise<"unavailable" | "blocked" | "denied" | "granted" | "limited"> {
```

Comment:
Can we change the case of permission to `PascalCase`? So, it would be shown to the user like "Unavailable", "Blocked", Etc..